test/e2e/cloud_manager/helper_test.go

```diff
@@ -392,3 +392,27 @@ func deleteProject(projectID string) error {
 	cmd.Env = os.Environ()
 	return cmd.Run()
 }
+
+const (
+	maxRetryAttempts   = 10
+	sleepTimeInSeconds = 30
+)
+
+func deleteProjectWithRetry(t *testing.T, projectID string) {
```

Comment:
We already have `deleteProjectWithRetry` in `atlas_e2e_test_generator_test`.go. Aren't we already using it?

Reply:
we can't use it here, it's scoped for atlas e2e tests and this is for CM/OM tests